src/extension/tools/node/getErrorsTool.tsx

```diff
@@ -62,14 +63,17 @@ class GetErrorsTool extends Disposable implements ICopilotTool<IGetErrorsParams>
 				throw new Error(`Invalid input path ${filePath}`);
 			}
 
-			let diagnostics = range
-				? findDiagnosticForSelectionAndPrompt(this.languageDiagnosticsService, uri, new Range(...range), undefined)
-				: this.languageDiagnosticsService.getDiagnostics(uri);
-
-			diagnostics = diagnostics.filter(d => d.severity <= DiagnosticSeverity.Warning);
+			let diagnostics: vscode.Diagnostic[] = [];
+			if (isJupyterNotebookUri(uri)) {
```

Comment:
Shouldn't this be used for all notebooks, not just Jupyter?
I think you can use `INotebookService.hasSupportedNotebook` to check if Uri is a notebook document